src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1182,4 +1182,15 @@ namespace winrt::TerminalApp::implementation
             args.Handled(handled);
         }
     }
+    void TerminalPage::_HandleToggleBroadcastInput(const IInspectable& /*sender*/,
+                                                   const ActionEventArgs& args)
+    {
+        if (const auto activeTab{ _GetFocusedTabImpl() })
+        {
+            activeTab->ToggleBroadcastInput();
+        }
+
+        args.Handled(true);

```

Comment:
We may want to only mark this as handled if there was a terminal tab (consider pressing Ctrl+B or whatever in the settings tab -- it should go through)